drivers/overlay/ov_network.go

```diff
@@ -601,6 +618,9 @@ func (n *network) initSubnetSandbox(s *subnet, restore bool) error {
 		}
 	} else {
 		if err := n.setupSubnetSandbox(s, brName, vxlanName); err != nil {
+			// The error in setupSubnetSandbox could be a temporary glitch. reset the
+			// subnet once object to allow the setup to be retried on another endpoint join.
+			s.once = &sync.Once{}
```

Comment:
@mavenugo The way restore works is fundamentally different. For the restore case we do handle something similar, but at the network level; https://github.com/docker/libnetwork/blob/master/drivers/overlay/overlay.go#L99

Going strictly by the code, yes, `restoreSubnetSandbox` can also fail. But all the recent issues reported for this error are in swarm mode (ie: not the restore code path). For the `restore` case we have to try with a forced error and verify things end to end. It belongs in a separate PR.